<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>724</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>908</width>
    <height>724</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>908</width>
    <height>724</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>811</width>
     <height>621</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Categorias Diversas</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Valor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Raça ou Serviço</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtValorCategoria">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>#####,##</string>
    </property>
    <property name="text">
     <string>,00</string>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlteracoes">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);
</string>
    </property>
    <property name="text">
     <string>Alterar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCachorro">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cachorro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExcluir">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);
</string>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCadastrar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Cadastrar raça ou serviço</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGato">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gato</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbServico">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Serviço</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListarCategoria">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);
</string>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelaCategorias">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 237, 255);</string>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string>Categorias</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtCadastrar</tabstop>
  <tabstop>txtValorCategoria</tabstop>
  <tabstop>rbServico</tabstop>
  <tabstop>rbGato</tabstop>
  <tabstop>rbCachorro</tabstop>
  <tabstop>btnCadastrar</tabstop>
  <tabstop>btnAlteracoes</tabstop>
  <tabstop>btnListarCategoria</tabstop>
  <tabstop>btnExcluir</tabstop>
  <tabstop>tabelaCategorias</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
